app/controllers/LegacyApiController.scala

```diff
@@ -452,4 +489,13 @@ class LegacyApiController @Inject()(annotationController: AnnotationController,
       } yield Ok(Json.toJson(newJson)).copy(header = result.header)
     } else Fox.successful(result)
 
+  private def logVersioned(request: SecuredRequest[WkEnv, _]): Unit =
+    logVersioned(request.identity, request.uri)
```

Comment:
What exactly is `request.identity`?

Reply:
`SecuredRequest[WkEnv]` is the request class constructed by Silhouette that enforces user authentication. It makes the requesting `User` (our own model, fetched from the db upon successful authentication) available as `request.identity`. The `UserAwareRequest[WkEnv]` is similar, but there the `request.identity` is an `Option[User]`